python/paddle/fluid/tests/unittests/mkldnn/test_mul_mkldnn_op.py

```diff
@@ -0,0 +1,193 @@
+# Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
move int8 related ut from `test_mul_mkldnn_op.py` to `test_mul_int8_mkldnn_op.py`